src/types/react-table-config.d.ts

```diff
@@ -0,0 +1,122 @@
+import { BoxProps } from '@chakra-ui/react'
```

Comment:
do we need this file if we have the `@types/react-table` package?

Reply:
yes their types doesn't include all the plugins, its weird this is how they recommend you do it with this version